content/events/ipfsFriendsCafeParis.md

```diff
@@ -11,4 +11,6 @@ tag: "PLN Event"
 dateTBD: true
 preferredContacts:
   - "email|mailto:miwa@protocol.ai"
----
\ No newline at end of file
+eventHosts:
+  - 'IPFS|<host_logo>'
```

Comment:
@laeccles Since an event host IPFS is already existing with a different logo, please confirm if the host here can be updated to IPFS Friends.

Reply:
@WinsCodes Per Ryo we would like the host to be listed as IPFS.  The host will not be IPFS Friends Cafe.

Reply:
@laeccles Great, so can you also please confirm if the IPFS' existing logo _(please refer below)_ be used _(instead of the one shared on this Pull request)_ since the event host is required to be IPFS - 

**IPFS' Prod-existing logo -** 
![image](https://github.com/memser-spaceport/pln-events/assets/120092872/eb1d5863-67f0-4e9f-9db0-fad7612750cb)

Reply:
@WinsCodes We would like to use the coffee cup logo per Ryo for this event.